src/Amazon.php

```diff
@@ -113,8 +115,21 @@ public static function get_available_shipping_methods(?\WC_Order $order) {
       return FALSE;
     }
 
+    if (!did_action('woocommerce_load_cart_from_session')) {
+      $GLOBALS['wp_actions']['woocommerce_load_cart_from_session'] = TRUE;
+    }
```

Comment:
As you're creating an actual session now, this hack shouldn't be necessary anymore?

Reply:
I will test it without this and remove it if needed.

Reply:
Jup, it worked well without this.